modules/ROOT/pages/apikit-graphql-api-implementation.adoc

```diff
@@ -0,0 +1,525 @@
+    longitude: Float
+}
+
+input AddressInput {
+    street: String
+    city: String
+    zipcode: String
+    latitude: Float
+    longitude: Float
+}
+
+input UserInput {
+    name: String
+    username: String
+    email: String
+    phone: String
+    website: String
+    address: AddressInput
+}
+
+type Query {
+    # Looks for a user by ID
+    getUser(id: ID!): User
+    # Looks for a post by ID
+    getPost(id: ID!): Post
+    # Returns the list of all users
+    allUsers: [User]
+    # Returns the list of all posts
+    allPosts: [Post]
+}
+
+type Mutation {
+    # Creates a new user
+    createUser(user: UserInput!): User!
+    # Creates a new post
+    createPost(userId: ID!, title: String!, body: String!): Post!
+    # Adds a comment to a post
+    postComment(userId: ID!, postId: ID!, body: String!): Comment!
+    # Adds a reply to a comment
+    replyToComment(commentId: ID!, userId: ID!, body: String!): Comment!
+}
+----
+
+== Request Queries Examples
+
+In the following example, the type *Query* defines the list of queries that the service exposes.
+
+----
+...
+type Query {
+    # Looks for a user by ID
+    getUser(id: ID!): User
+    # Looks for a post by ID
+    getPost(id: ID!): Post
+    # Returns the list of all users
+    allUsers: [User]
+    # Returns the list of all posts
+    allPosts: [Post]
+}
+...
+----
+
+=== Retrieve All the Posts
+
+This use case makes a simple request for the data of each existing post.
+ 
+==== Make a Query
+
+----
+{
+   allPosts {
+       id
+       title
+       body
+   }
+}
+----
+
+==== Trigger a Request
+
+----
+curl --location --request POST 'http://localhost:8081/graphql' \
+--header 'Content-Type: application/json' \
+--data-raw '{"query":"{allPosts{id title body}}","variables":{}}'
+----
+
+==== Get a Response
+
+----
+{
+   "data": {
+       "allPosts": [
+           {
+               "id": "88",
+               "title": "tellus nisi eu orci mauris lacinia sapien quis libero",
+               "body": "Praesent id massa id nisl venenatis lacinia. Aenean sit amet justo. Morbi ut odio.\n\nCras mi pede, malesuada in, imperdiet et, commodo vulputate, justo. In blandit ultrices enim. Lorem ipsum dolor sit amet, consectetuer adipiscing elit.\n\nProin interdum mauris non ligula pellentesque ultrices. Phasellus id sapien in sapien iaculis congue. Vivamus metus arcu, adipiscing molestie, hendrerit at, vulputate vitae, nisl."
+           },
+           {
+               "id": "89",
+               "title": "mauris lacinia sapien quis libero",
+               "body": "Sed ante. Vivamus tortor. Duis mattis egestas metus.\n\nAenean fermentum. Donec ut mauris eget massa tempor convallis. Nulla neque libero, convallis eget, eleifend luctus, ultricies eu, nibh.\n\nQuisque id justo sit amet sapien dignissim vestibulum. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nulla dapibus dolor vel est. Donec odio justo, sollicitudin ut, suscipit a, feugiat et, eros."
```

Comment:
See prev q

Reply:
As this is an example of a response, I guess we can get rid of the \n\n and create a single paragraph.
Let's wait for the dev team comments.